### ISSUE TITLE
Narrator does not announce the state details for combobox collapse/expanded

### ISSUE THREAD

**Issue by sereymonyros** [microsoft/fluentui#16105]
https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox

Step to repro:
1) get the focus on the combobox
2) hit enter/space to open the contextmenu

Actual:
screen reader doesn't announce the state details for combobox collapse/expanded

Expected
screen reader should announce the state details, so user is able to know/hear the state

**Comment by JustSlone** [comment]
@sereymonyros thanks for filing this. We're actually already tracking this over here #13926

Closing out this issue.
